sql/glam-fenix-dev/glam_etl/firefox_desktop_glam_nightly__histogram_bucket_counts_v1/query.sql

```diff
@@ -98,6 +98,14 @@ distribution_metadata AS (
   FROM
     UNNEST(
       [
+        STRUCT(
```

Comment:
Lots of changes because of reordering due to changing from `list` to `set` in order to avoid duplicates that might come from both `fenix` and `firefox-desktop`